UM-USElab/gradecraft-development
# 3121 quick grade style refactor

### Status
READY

### Description
The goal of this PR is to allow the dynamic changing of the quick grade styling from the mass edit page on individually graded assignments.

**At the moment, we will also need to refactor the group grading form as well in order to allow this capability.** See issue #3321 

In order to accomplish this, the mass edit form was rewritten in Angular along with the team selector filter that was previously rendered with it.

Refer to #2956 for previously discussed details on available quick grading options.

### Gem dependencies
N/A

### Migrations
NO

### Steps to Test or Reproduce

- Ensure that the students in the table are updated when the team filter selection changes
- Ensure that mass editing of grades works as expected for all individual assignment types

### Impacted Areas in Application
Mass grading

======================
Closes #3121
